crates/rust-analyzer/Cargo.toml

```diff
@@ -91,6 +91,7 @@ xshell.workspace = true
 test-utils.workspace = true
 test-fixture.workspace = true
 syntax-bridge.workspace = true
+mktemp = "0.5"
```

Comment:
I don't think the dependency is worth the test. We also generally don't test r-a commands this way, by running them externally. Yes, this can be problems from this, but the cost of testing this is too high.